back-end/API/app/repositories/enterprise_repo.py

```
@@ -1,96 +1,79 @@
+            db.select(Enterprise)
+            .join(Enterprise.users)
+            .where(User.id == user_id)
+        ).scalar_one_or_none()
+        if enterprise is None:
+            raise NotFoundException(f"Enterprise not found with user id", user_id)
+        return EnterpriseReadDTO.model_validate(enterprise)
+
+    def update(self, dto: EnterpriseUpdateDTO) -> EnterpriseReadDTO:
+        enterprise = Enterprise.query.filter_by(id=dto.id).options(db.joinedload(Enterprise.users)).first()
+        if enterprise is None:
+            raise NotFoundException("Enterprise not found", dto.id)
```

Ruff (0.15.15)

[warning] Avoid specifying long messages outside the exception class

(TRY003)